Essay_Analysis_Tool/Interface/ILoggerView.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Essay_Analysis_Tool.Business;
+using FastColoredTextBoxNS;
+
+namespace Essay_Analysis_Tool.Interface
+{
+    public interface ILoggerView
+    {
+        /// <summary>
+        /// Appends the given text to the FastColoredTextBox instance.
+        /// </summary>
+        /// <param name="text">Message text.</param>
+        /// <param name="style">Style of message.</param>
+        void AppendText(string text, Style style);
+
+        /// <summary>
+        /// Sets the controller for the <see cref="LoggerView"/>.
+        /// </summary>
+        /// <param name="controller">Controller object.</param>
+        void SetController(LoggerViewController controller);
+
+        /// <summary>
+        /// Shows the <see cref="LoggerView"/>.
+        /// </summary>
+        void ShowView();
```

Comment:
Just rename it to Show because when You want to call just `_view.Show();` and not `_view.ShowView();` :)